src/main/scala/io/pinecone/spark/pinecone/PineconeOptions.scala

```diff
@@ -4,18 +4,13 @@ import PineconeOptions._
 import org.apache.spark.sql.util.CaseInsensitiveStringMap
 
 class PineconeOptions(config: CaseInsensitiveStringMap) extends Serializable {
-  private val DEFAULT_BATCH_SIZE = 100
+  private val DEFAULT_BATCH_SIZE = 1000
```

Comment:
Optimal batching strategy is creating a large batch of vectors while satisfying the following two conditions:
1. 1000 vectors  
2. The packet size must be <2MB

Reply:
We will update the batch vector size to 1000 in the follow up PR